application/console/cards_console.rb

```diff
@@ -0,0 +1,157 @@
+    return unless yes?
+
+    @account.cards.delete(chosen_card)
+  end
+
+  def put_account_money(command)
+    operation = validate_operation(command)
+    return if operation.nil?
+
+    amount, card = card_amount(operation)
+    return output(I18n.t('ERROR.tax_higher')) unless card.operation_put_valid?(amount)
+
+    card.put_money(amount)
+    output(I18n.t('common_phrases.after_put',
+                  amount: amount,
+                  number: card.number,
+                  balance: card.balance,
+                  tax: card.put_tax(amount)))
+  end
+
+  def validate_operation(command)
+    action = COMMANDS.key(command)
+    output(I18n.t("operations.choose_card.#{action}"))
+    chosen_card = select_card
+    return if chosen_card.nil?
+
+    output(I18n.t("operations.amount.#{action}"))
+    amount = validate_amount
+    return if amount.nil?
+
+    { chosen_card: chosen_card, amount: amount }
+  end
+
+  def validate_amount
+    amount = user_input.to_i
+    amount.positive? ? amount : output(I18n.t('ERROR.correct_amount'))
+  end
+
+  def withdraw_account_money(command)
+    operation = validate_operation(command)
+    return if operation.nil?
+
+    amount, card = card_amount(operation)
+    return output(I18n.t('ERROR.no_money_left')) unless card.operation_withdraw_valid?(amount)
+
+    card.withdraw_money(amount)
+    output(I18n.t('common_phrases.after_withdraw',
+                  amount: amount,
+                  number: card.number,
+                  balance: card.balance,
+                  tax: card.withdraw_tax(amount)))
+  end
+
+  def send_account_money(command)
+    operation = validate_operation(command)
+    return if operation.nil?
+
+    amount, sender_card = card_and_amount(operation)
+    output(I18n.t('common_phrases.recipient_card'))
+
+    recipient_card = recipient_card_validation
+    return if recipient_card.nil?
+
+    return unless validate_send_operation_taxes(sender_card, recipient_card, amount)
+
+    send_money_operation(sender_card, recipient_card, amount)
+  end
+
+  def validate_send_operation_taxes(sender_card, recipient_card, amount)
+    return output(I18n.t('ERROR.no_money_left')) unless sender_card.operation_send_valid?(amount)
+    return output(I18n.t('ERROR.no_money_on_recipient')) unless recipient_card.operation_put_valid?(amount)
+
+    true
+  end
+
+  def send_money_operation(sender_card, recipient_card, amount)
+    sender_card.send_money(amount)
+    recipient_card.put_money(amount)
+    output(I18n.t('common_phrases.after_withdraw',
+                  amount: amount,
+                  number: sender_card.number,
+                  balance: sender_card.balance,
+                  tax: sender_card.sender_tax(amount)))
+    output(I18n.t('common_phrases.after_put',
+                  amount: amount,
+                  number: recipient_card.number,
+                  balance: recipient_card.balance,
+                  tax: recipient_card.put_tax(amount)))
```

Comment:
let's create a method that will return this hash because this logic is repeated in several places

Reply:
created method and put it to return hash in this method and in line 39. the same i did in line 68 and 96.

Reply:
redone this method and it is probably what you asked me to do.